test_non_rpm_files.py

```diff
@@ -0,0 +1,38 @@
+"""
+#
+#   This document is used to test
+#   Non RPM files in the system.
+#
+#
+"""
+
+import os
+import unittest
+
+from tunirtests.testutils import system
```

Comment:
`from tunirtests.testutils import system` ? It should be `from .testutils import system` no?